packages/cursorless-vscode/src/ide/vscode/VscodeOpenLink.ts

```diff
@@ -17,7 +19,10 @@ export default async function vscodeOpenLink(
   }
 
   if (filteredLinks.length === 0) {
-    return false;
+    let commandId = options?.openInSplit
+      ? "editor.action.revealDefinitionAside"
+      : "editor.action.revealDefinition";
+    await vscode.commands.executeCommand(commandId);
```

Comment:
I don't think this will actually work. You are executing reveal definition on the current selection always and not on the actual target. I would probably still use our own `actions.executeCommand`

Reply:
I think you're right. I didn't change this though-don't really understand what you mean by 'use our own actions.executeCommand'-can you help me understand?

Reply:
The one you removed in this pull request

https://github.com/cursorless-dev/cursorless/blob/766864d9fa979124283b236bbb4738204a13fe45/packages/cursorless-
engine/src/actions/FollowLink.ts#L27-L31

But since we are already setting selection for the `editor.action.openLink` command it's probably best to merge these behaviors

Reply:
oh okay I see it now

Reply:
I wasn't sure exactly what you meant but I gave it a shot; let me know how it looks. Thanks!

Reply:
You still need the if statement to check if we actually have found a link. The set selection code could be extracted to ear separate function that is used in both cases.
Also I would remove the return boolean type and just have void.

Reply:
1. Extracted the set selection code
2. the return type is harder-that type is defined in the interface. Are you sure I should remove it?